dashboard/src/components/Table/TreeTable.tsx

```diff
@@ -50,7 +52,9 @@ const TreeTableRow = (row: TreeTableBody): JSX.Element => {
       <TableCell>
         {sanitizeTableValue(row.tree_names.join(', '), false)}
       </TableCell>
+      <TableCell>{sanitizeTableValue(row.branch, false)}</TableCell>
       <TableCell>{sanitizeTableValue(row.commit)}</TableCell>
+      <TableCell>{sanitizeTableValue(row.date.split('T')[0])}</TableCell>
```

Comment:
Array indexes on typescript can return undefined and the linter won't warn us unless we have a rule for that, maybe add a validation here